alinka_website.tf

```diff
@@ -0,0 +1,21 @@
+resource "aws_iam_user" "alinka_website" {
+  name = "alinka_website"
+}
+
+resource "aws_iam_access_key" "alinka_website" {
+  user = aws_iam_user.alinka_website.name
+}
+
+resource "aws_route53_zone" "alinka_website" {
+    name = "alinka.io"
```

Comment:
I've just built a new Alinka website and it still doesn't look like something to be deployed to production. Can we for the time being put it on `dev.alinka.io` domain?